media.css

```diff
@@ -0,0 +1,364 @@
+@media screen and (min-width: 1140px) {
```

Comment:
- [ ] As requested by the previous reviewer. Please make sure you fix your design it breaks from `768px` to `1140px` 🙏I suggest you use `min-width: 768px`